web/modules/custom/farm_page/src/Services/FarmPageService.php

```diff
@@ -108,11 +130,23 @@ public function getProductsInfomation($products_array) {
       $product['product_description'] = $product_description[0]['value'];
       // Get Category.
       $product_category_id = $product_in_array->get('field_category')->getValue();
-      $product_category_name = Term::load($product_category_id[0]['target_id']);
-      $product['product_category'] = $product_category_name->getName();
-      array_push($products, $product);
+      $product_category = $this->getTaxonomyTerm($product_category_id);
+      $product['product_category'] = $product_category;
```

Comment:
$product['category'] = $this->getTaxonomyTerm($term->tid);

Reply:
fixed